config/vmagent-prometheus.yml.erb

```diff
@@ -24,3 +24,6 @@ scrape_configs:
   - job_name: <%= ENV['VMAGENT_APP_JOB_NAME'] || 'web' %>
     static_configs:
     - targets: ['localhost:<%= ENV['PORT'] %>']
+    basic_auth:
+      username: <%= ENV['PROMETHEUS_METRICS_USERNAME'] %>
+      password: <%= ENV['PROMETHEUS_METRICS_PASSWORD'] %>
```

Comment:
Let's namespace these under `VMAGENT_` like the others, perhaps `VMAGENT_SCRAPE_USERNAME` and `VMAGENT_SCRAPE_PASSWORD`?

Reply:
Done! 👍